packages/js/src/ui/components/elements/Bell/Bell.tsx

```diff
@@ -1,24 +1,32 @@
 import { createSignal, JSX, onCleanup, onMount } from 'solid-js';
+import { requestLock } from '../../../helpers/browser';
+import { NV_INBOX_WEBSOCKET_LOCK } from '../../../helpers/constants';
 import { useNovu } from '../../../context';
 import { BellContainer } from './DefaultBellContainer';
+import { useTabsChannel } from '../../../helpers/useTabsChannel';
 
 type BellProps = {
   children?: ({ unreadCount }: { unreadCount: number }) => JSX.Element;
 };
 export function Bell(props: BellProps) {
   const [unreadCount, setUnreadCount] = createSignal(0);
   const novu = useNovu();
+  const { postMessage } = useTabsChannel<number>({ onMessage: setUnreadCount });
```

Comment:
The helper hook allows sending a message across multiple tabs. It's based on the `BroadcastChannel` API. The message will be only received by the other tabs but not the sender.